src/main/java/uos/dev/restcli/HttpRequestFilesExecutor.kt

```diff
@@ -73,6 +73,12 @@ class HttpRequestFilesExecutor constructor(
             parser.parse(httpFilePath)
         } catch (e: Exception) {
             logger.error(e) { "Can't parse $httpFilePath" }
+            val trace = TestGroupReport.Trace(
+                httpTestFilePath = httpFilePath,
+                scriptHandlerStartLine = -1
+            )
+            TestReportStore.addTestGroupReport("-", trace)
+            TestReportStore.addTestReport("Parsing",false, e.message,e.message)
```

Comment:
Please format code before commit.
For example,
`"Parsing",false` => `"Parsing", false`